src/viz-parole-revocation/VizParoleRevocation.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Measure from "react-measure";
+import styled from "styled-components";
+import { DIMENSION_KEYS, TOTAL_KEY } from "../constants";
+import BubbleChart from "../bubble-chart";
+
+const HEIGHT = 450;
+
+const VizParoleRevocationContainer = styled.div`
```

Comment:
hmm yeah interesting, that's a good idea to make the other component private. 

I guess if pressed I would say that a "container" is smart (e.g. does data stuff) and a "wrapper" is dumb, because that is how the word "container" has been co-opted in React Land. I haven't been following that convention but I think it's probably a good idea and we should? (i.e. private styled components are 'wrappers', full-on React components that pass through or modify props are "containers")

Reply:
I dig the container/wrapper distinction